builder-support/debian/authoritative/debian-buster/control

```diff
@@ -5,7 +5,6 @@ Priority: optional
 Standards-Version: 4.5.1
 Build-Depends: bison,
                curl,
-               debhelper-compat (= 10),
```

Comment:
would be nicer if it said `debhelper (>= 10)`

Reply:
done, thanks!